Source/WebKit/UIProcess/ios/forms/WKDateTimeInputControl.mm

```diff
@@ -192,6 +228,14 @@ - (NSString *)_sanitizeInputValueForFormatter:(NSString *)value
     return value;
 }
 
+- (RetainPtr<NSISO8601DateFormatter>)iso8601DateFormatterForCalendarView
+{
+    auto dateFormatter = adoptNS([[NSISO8601DateFormatter alloc] init]);
+    [dateFormatter setTimeZone:[_calendarView timeZone]];
+    dateFormatter.get().formatOptions = NSISO8601DateFormatWithYear | NSISO8601DateFormatWithWeekOfYear | NSISO8601DateFormatWithDashSeparatorInDate;
```

Comment:
I think we usually do `[dateFormatter setFormatOptions:` instead of `.get().formatOptions`, but it doesn't really matter

Reply:
Yes, please use bracket syntax here.